samples/insert-query-with-get-trasanction.js

```diff
@@ -0,0 +1,79 @@
+// Copyright 2024 Google LLC
```

Comment:
Why are we adding samples here. Samples normally need to be discussed with documentation team and tags needs to be discussed with them. 
We recommend customers to use `runTransaction` method.

Reply:
I have removed the samples, as discussed